package.json

```diff
@@ -19,9 +19,14 @@
     "@redhat-cloud-services/host-inventory-client": "^1.0.116",
     "@redhat-cloud-services/vulnerabilities-client": "^1.2.10",
     "@scalprum/react-core": "^0.1.9",
+    "@testing-library/jest-dom": "^6.1.4",
+    "@testing-library/react": "^12.0.0",
+    "@testing-library/user-event": "^14.5.1",
```

Comment:
Maybe it would be better to keep this one also in the version 12 range. If it works it should be fine, but there could be issues and if there are, do try to downgrade this one as well.